src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/PostEntity.java

```diff
@@ -2,12 +2,14 @@
 
 import com.google.common.base.Predicate;
 import com.google.common.collect.Iterables;
+import com.wikia.webdriver.common.core.WikiaWebDriver;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.WikiaWebDriver'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)